code/src/Core/Locations/RemoteTemplatesSource.cs

```diff
@@ -63,11 +72,9 @@ private static void DownloadContent(string sourceUrl, string file)
             }
             catch (Exception ex)
             {
-                string msg = "The templates content can't be downloaded.";
-
-                AppHealth.Current.Exception.TrackAsync(ex, msg).FireAndForget();
-
-                throw;
+                string msg = $"Templates content can't be downloaded right now, we will try it later.";
```

Comment:
This is a blocking bug right now.  I hear you but disagree as that is adding in additional work here.